app/assets/javascripts/analytics/static-analytics.js

```diff
@@ -125,7 +125,7 @@
   };
 
   StaticAnalytics.prototype.stripPII = function (value) {
-    return this.analytics.stripPII(value)
+    return this.analytics.stripPII.stripPII(value)
```

Comment:
I assume this is going to get renamed next?

Reply:
I'd suggest naming the module just PII now actually.

Reply:
Renamed. It's still a bit confusing though.